sdk/maps/azure-maps-timezone/azure/maps/timezone/_timezone_client.py

```diff
@@ -0,0 +1,289 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import Union, Any, List
+from azure.core.credentials import AzureKeyCredential, TokenCredential
+from azure.core.pipeline.policies import AzureKeyCredentialPolicy
+from azure.core.tracing.decorator import distributed_trace
+from .models import LatLon
+from ._generated import TimezoneClient as _MapsTimezoneClient
+from ._generated.models import (
+    IanaId,
+    TimezoneIanaVersionResult,
+    TimezoneResult,
+    TimezoneWindows
+)
+from ._version import VERSION
+
+
+# To check the credential is either AzureKeyCredential or TokenCredential
+def _authentication_policy(credential):
+    authentication_policy = None
+    if credential is None:
+        raise ValueError("Parameter 'credential' must not be None.")
+    if isinstance(credential, AzureKeyCredential):
+        authentication_policy = AzureKeyCredentialPolicy(
+            name="subscription-key", credential=credential
+        )
+    elif credential is not None and not hasattr(credential, "get_token"):
+        raise TypeError(
+            f"Unsupported credential type: {type(credential)}. "
+            "Use an instance of AzureKeyCredential or a token credential from azure.identity"
+        )
+    return authentication_policy
+
+
+class MapsTimezoneClient:  # pylint: disable=client-accepts-api-version-keyword
+    """Azure Maps Time Zone REST APIs.
+
+    :param credential: Credential needed for the client to connect to Azure. Required.
+    :type credential: ~azure.core.credentials.AzureKeyCredential or ~azure.core.credentials.TokenCredential
+    :keyword client_id: Specifies which account is intended for usage in conjunction with the Azure
+     AD security model.  It represents a unique ID for the Azure Maps account and can be retrieved
+     from the Azure Maps management  plane Account API. To use Azure AD security in Azure Maps see
+     the following `articles <https://aka.ms/amauthdetails>` for guidance. Default value is None.
+    :type client_id: str
```

Comment:
Recommend to change the type for `keyword` args to `paramtype`

Reply:
You can refer here for more recommendation on [method docstring](https://eng.ms/docs/experiences-devices/webxt/microsoft-maps/azure-maps/azure-maps/azure-maps-team-documentation/developer/sdks/rest/python/pythondocstring#method-docstrings)